cms/templates/index.html

```diff
@@ -589,8 +605,8 @@ <h3 class="title title-3">${_("Can I create courses in {studio_name}?").format(s
         <p>${_("Your request to author courses in {studio_name} has been denied. Please {link_start}contact {platform_name} Staff with further questions{link_end}.").format(
             studio_name=settings.STUDIO_NAME,
             platform_name=settings.PLATFORM_NAME,
-            link_start=help_link_start,
-            link_end=help_link_end,
+            link_start=HTML(help_link_start),
```

Comment:
1. Get rid of the `help_link_start` and `help_link_end` variables and inline here.
2. Have the format call outside of the HTML() call so that the variable gets properly escaped:
```
link_start=HTML('<a href="mailto:{email}">').format(email=settings.TECH_SUPPORT_EMAIL),
```